documentation/book/con-loggers-kafka-bridge.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-kafka-bridge.adoc
+
+[id='con-loggers-kafka-bridge-{context}']
+
+= Configuring loggers for the {ProductName} Kafka Bridge
+
+The {ProductName} Kafka bridge allows you to set a different log level for each operation that is defined by the related OpenAPI specification.
+
+Each operation has a corresponding API endpoint through which the bridge receives requests from HTTP clients.
+You can change the log level on each endpoint to produce more or less fine-grained logging information about the incoming and outgoing HTTP requests.
+
+Loggers are defined in the `log4j.properties` file, which has the following default configuration for `healthy` and `ready` endpoints:
```

Comment:
You should probably add how to configure `log4j.properties` using the external logging configuration. Because this will not be clear to users.